### DISCUSSION TITLE
Is it possible to get the max number of user processes?

### DISCUSSION THREAD

**Discussion by Claudenw** [oshi/oshi#2495] [Q&A] [answered]
I am looking for a way to retrieve the value from `ulimit -u` on a linux system.  Is this possible in OSHI?

**Comment by dbwiddis** [comment, marked as answer]
Looking at the man page, `-u` "Specifies how many processes a user can create".  That's not part of OSHI's API.  It does offer a "max file descriptors" (`ulimit -n` equivalent) but that's clearly different.

There is an `ExecutingCommand` util class that will let you run any command line and get the result, which would serve your purpose.

**Comment by Claudenw** [comment]
Thanks for the quick response.
